test/manual/amp-lightbox-gallery-launch.amp.html

```diff
@@ -179,7 +171,8 @@ <h2>AMP-YOUTUBE Layout Test</h2>
     <amp-youtube width="480"
       height="270"
       layout="responsive"
-      data-videoid="lBTCB7yLs8Y">
+      data-videoid="lBTCB7yLs8Y"
+      lightbox-thumbnail-id="fb-thumbnail-img">
```

Comment:
should this be `yt-thumbnail-img`?

Reply:
Yup! Thanks for the catch!

Reply:
Oh actually this doesn't matter as it is test code. But might as well change it to proper naming anyway.